.github/workflows/ide-publish.yml

```diff
@@ -21,12 +21,12 @@ jobs:
 
       - name: Build and Publish Docker Image
         env:
-          CONTAINER_IMAGE: ghcr.io/eclipse-researchlabs/smartclide/smartclide-ide:${{ steps.get_tag_name.outputs.TAG_NAME }}
+          CONTAINER_IMAGE: ghcr.io/unparallel-innovation/smartclide-ide:${{ steps.get_tag_name.outputs.TAG_NAME }}
```

Comment:
This should be renamed to ghcr.io/opensmartclide/smartclide/smartclide-ide, so that our deployment scripts are able to retrieve them

Reply:
I have approved the changes in [#17](https://github.com/eclipse-opensmartclide/smartclide-ide-front-end/pull/17), which should unlock this.

Reply:
now you should rebase this one with main/master